ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -7,40 +7,45 @@ function computeRecleanFeatures(StudyFolder, ...
                                 CorticalParcellationFile, ...
                                 SubRegionParcellationFile, ...
                                 WMLabelFile, ...
-                                CSFLabelFile)
+                                CSFLabelFile, VisualAreasFile, LanguageAreasFile, SubRegionsFile, NonGreyParcelsFile)
 
 HCPPIPEDIR = getenv('HCPPIPEDIR')
+FullGreyordinateLength = 91282; 
 
 fMRINames = myreadtext(fMRIListName);
 RunsXNumTimePoints = str2double(RunsXNumTimePoints);
 ResolutionNum = str2double(Resolution);
 
 CorticalParcellation=ciftiopen(CorticalParcellationFile,'wb_command');
 
-VisualAreas=[1 2 3 4 5 6 7 13 16 17 18 19 20 21 22 23 48 49 142 152 153 154 156 158 159 160 163 181 182 183 184 185 186 187 193 196 197 198 199 200 201 202 203 228 229 322 332 333 334 336 338 339 340 343];
+VisualAreas=get_ROI_from_txt(VisualAreasFile); % VisualAreasFile HCPpipelines/global/config/Visual.ROI.txt
```

Comment:
I meant for the bash script to make a cifti file and pass that to matlab (so that matlab doesn't need to do any cifti index manipulation), but I guess at least the list isn't hardcoded in matlab anymore.